### PR TITLE
fix(fonts): ubermove scale updates

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: src/themes/light-theme-with-move.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        fontFamily: primaryFontFamily,
+        fontSize: '12px',
+        fontWeight: 'normal',
+        lineHeight: '20px',
+      },
+      font250: {
+        fontFamily: primaryFontFamily,
+        fontSize: '12px',
+        fontWeight: 'bold',
+        lineHeight: '20px',
```
Comment (nadiia) [inline src/themes/light-theme-with-move.js]: these are the same as in the default theme, no need to override

File: src/themes/light-theme-with-move.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      },
+      font400: {
+        fontFamily: primaryFontFamily,
+        fontSize: '16px',
+        fontWeight: 'normal',
+        lineHeight: '24px',
+      },
+      font425: {
```
Comment (nadiia) [inline src/themes/light-theme-with-move.js]: seems like this one should be the 450 font, and 20/28 - 500

File: src/themes/light-theme-with-move.js (hunk 1 above)
Comment (gergelyke) [reply src/themes/light-theme-with-move.js]: yeah, I agree - the reason I still added them here is for clarity, to make sure who wants to quickly understand what fonts the UberMoveTheme is using will know instantly

does it make sense?

Comment (nadiia) [reply src/themes/light-theme-with-move.js]: yeah, makes sense

File: src/themes/light-theme-with-move.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      },
+      font900: {
+        fontFamily: primaryFontFamily,
+        fontSize: '52px',
+        fontWeight: 'bold',
+        lineHeight: '68px',
+      },
+      font1000: {
+        fontFamily: primaryFontFamily,
+        fontSize: '92px',
```
Comment (nadiia) [inline src/themes/light-theme-with-move.js]: this should be `72px`

File: src/themes/light-theme-with-move.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+const primaryFontFamily =
+  'UberMoveText, "Open Sans", "Helvetica Neue", Helvetica, sans-serif';
+
+const secondaryFontFamily =
+  'UberMove, UberMoveText, "Open Sans", "Helvetica Neue", Helvetica, sans-serif';
+
+export const LightThemeMove = createTheme(
+  {
+    ...lightThemePrimitives,
+    primaryFontFamily,
```
Comment (nadiia) [inline src/themes/light-theme-with-move.js]: do we need to pass `secondaryFontFamily ` in here?

Comment (gergelyke) [reply src/themes/light-theme-with-move.js]: we don't - the factory has no concept of the `secondaryFontFamily`

**Pull request merged**